packages/api/endpoints/rules.js

```diff
@@ -87,6 +87,10 @@ async function post(req, res) {
   const name = apiRule.name;
 
   try {
+    if (await rulePgModel.exists(knex, { name })) {
```

Comment:
:stare: 

I'm going to come back to this, but we should determine if this was added as a P2 bugfix post merge, or why this wasn't already done in the P3 branch.   It's probably fine, but worth a double check.   Skipping for now, will revisit.